backend/apps/ifc_validation/tasks.py

```diff
@@ -507,7 +538,14 @@ def prerequisites_subtask(self, prev_result, id, file_name, *args, **kwargs):
             task.mark_as_failed(err)
             raise
 
-        if (proc.returncode is not None and proc.returncode != 0) or (len(proc.stderr) > 0):
+        # @nb previously we also checked for:
+        # or (len(proc.stderr) > 0):
+        #
+        # but I now get warnings:
+        #
+        # - features/environment.py:86: ContextMaskWarning: user code is masking context attribute 'gherkin_outcomes'; 
+        #                                                   see the tutorial for what this means
```

Comment:
I also noticed this warning a while ago, it appears every run in between scenario's. It seems it is because we set 
```
old_outcomes = getattr(context, 'gherkin_outcomes', []) 
context.gherkin_outcomes = old_outcomes
```
I think it's some leftover from the past, when had issues with context resetting/overlap between scenario's. The lines don't affect the outcomes that are being outputted to the tests or stored in the database. When I remove them, the warning goes away and nothing changes when running pytest or checking the platform.
It's probably safe and clearer to delete them ..